src/test/transaction_tests.cpp

```diff
@@ -180,6 +183,9 @@ BOOST_AUTO_TEST_CASE(tx_valid)
 
 BOOST_AUTO_TEST_CASE(tx_invalid)
 {
+    //UNIT-E TODO: Re enable the test after having properly changed tx_invalid.json
```

Comment:
I'd very much be in favor in having these tests fixed in this PR too.

Should it not be relatively straight forward to convert these json files?

We have a bunch of block parsers lying around which could be adapted for the needs, i guess:

- https://github.com/dtr-org/unit-e-tooling/tree/master/block-parsing
- https://github.com/dtr-org/bitcoinschema

Reply:
Of course, it's not a problem to generate yet another bunch of serialized transactions.
But this file - `tx_valid.json` is a collection of weird cases from real bitcoin chain, and I have some doubts that it's easy to reproduce them in a short time. Same for `tx_invalid.json`.
Moreover, I'm not going to merge this PR in the current state.